src/platforms/javascript/common/session-replay/configuration.mdx

```diff
@@ -87,21 +88,57 @@ new Replay({
 
 <Note>Captured bodies will be truncated to 150k characters max.</Note>
 
-## Mutation Limits
+## Identifying Users
 
-Session Replay works by recording incremental DOM changes that occur in your web application. Generally these changes occur in small batches and cause minimal overhead. However, it can be easy to overlook cases that will cause a large amount of DOM mutations to happen in a single update. An example is a custom dropdown component that has an unbounded list of options to render. This can negatively impact performance without Session Replay and its effects will be magnified with Session Replay enabled. In order to avoid this scenario, Session Replay will stop recording if it detects a large number of mutations (default: 10,000), which can be configured by setting `mutationLimit`. Additionally, we provide breadcrumbs in the replay to warn you when a large number of mutations are detected (default: 750).
+You can use the Sentry SDK to link a user to a session. Read more about it in our <PlatformLink to="/enriching-events/identify-user">docs</PlatformLink>.
+
+```javascript
+Sentry.setUser({ email: "jane.doe@example.com" });
+```
+
+## Using a Custom Compression Worker
+
+By default, Session Replay will use an inlined web worker script to compress Replay data before we send it over the network. This helps to drastically reduce the amount of data transferred, improving performance and reducing network overhead. Because compressing data is CPU intensive, we use a web worker to offload this work to a separate thread.
+
+While an inlined worker will work well for most applications, there are two main problems with this:
+
+1. Since the worker code is inlined, it increases the main bundle size of your application.
+2. Using inline workers can cause CSP violations if your application has a very strict CSP policy. [Learn more about configuring your CSP policy.](../#content-security-policy-csp)
```

Comment:
Does a PlatformLink work? They're more robust than ../ if page structure changes in the future.

```suggestion
2. Using inline workers can cause CSP violations if your application has a very strict CSP policy. See our <PlatformLink to="/session-replay/#content-security-policy-csp">CSP</PlatformLink> docs for more information.
```